src/commands/install.yml

```diff
@@ -60,11 +61,18 @@ steps:
 
         # FUNCTIONS
 
+        get_node_index () {
+          curl -Ls 'https://nodejs.org/dist/index.tab' | tail -n +2 | cut -f 1,3,10
+        }
+
         get_node_version () {
           if [[ <<parameters.node-version>> == "" || <<parameters.node-version>> == "latest" || <<parameters.node-version>> == "current" ]]; then
-            NODE_ORB_INSTALL_VERSION=$(curl -Ls -o /dev/null -w %{url_effective} \
-              "https://github.com/nodejs/node/releases/latest" | sed 's:.*/::' | cut -d 'v' -f 2)
+            NODE_ORB_INSTALL_VERSION=$(get_node_index | awk "NR<=1" | cut -f 1 | grep -E -o '[^v].*')
             echo "Latest version of Node.js is $NODE_ORB_INSTALL_VERSION"
+          elif [[ <<parameters.node-version>> == "lts" || <<parameters.node-version>> == "stable" ]]; then
+            # Codename is last field, first one with a name is newest lts
+            NODE_ORB_INSTALL_VERSION=$(get_node_index | grep -E "\t[a-zA-Z]+\$" | awk "NR<=1" | cut -f 1 | grep -E -o '[^v].*')
```

Comment:
@michaelgmcd It looks like this passes on macos (https://app.circleci.com/pipelines/github/CircleCI-Public/node-orb/461/workflows/9a88f65c-95fb-4b4e-a308-5e8a0ea62632/jobs/3201) but not on linux (https://app.circleci.com/pipelines/github/CircleCI-Public/node-orb/461/workflows/9a88f65c-95fb-4b4e-a308-5e8a0ea62632/jobs/3199), would you mind taking a look?

Reply:
Tried investigating, but the only log is "Exited with status code 1"

Is there a way we could get more error information?

Reply:
I think the issue was with the `\t` character used in the grep expression. I managed to resolve the build error by modifying the expression.